docs/environment/storage.md

```diff
@@ -76,3 +76,110 @@ Note that Redis and Memcache through ElastiCache run even when not used and thus
 pay-per-request mode where you don't pay when the store is not used. There is a package implementing the PSR cache interfaces using DynamoDB ([rikudou/psr6-dynamo-db](https://github.com/RikudouSage/DynamoDbCachePsr6)).
 
 This solution is ideal for cache data that can change during the life of the application (e.g. caching a website menu, an API response).
+
+#### Deploy DynamoDB table as part of serverless
+
+The example will be based on the minimal configuration which is as follows:
+
+```yaml
+service: app
+provider:
+    name: aws
+    runtime: provided.al2
+plugins:
+    - ./vendor/bref/bref
+functions:
+    app:
+        handler: index.php
+        layers:
+            - ${bref:layer.php-74-fpm}
+        events:
+            - httpApi: '*'
+```
+
+As any service, the DynamoDB table can be deployed using the `resources.Resources` subkey:
+
+```yaml
+service: app
+provider:
+    name: aws
+    runtime: provided.al2
+plugins:
+    - ./vendor/bref/bref
+functions:
+    app:
+        handler: index.php
+        layers:
+            - ${bref:layer.php-74-fpm}
+        events:
+            - httpApi: '*'
+resources:
+    Resources:
+        CacheTable:
+            Type: AWS::DynamoDB::Table
+            Properties:
+              AttributeDefinitions:
+                - AttributeName: id
+                  AttributeType: S
+              BillingMode: PAY_PER_REQUEST
+              TimeToLiveSpecification:
+                AttributeName: ttl
+                Enabled: true
+              KeySchema:
+                - AttributeName: id
+                  KeyType: HASH
+```
+
+Explanation of the properties:
```

Comment:
Not sure we need this much information here ? Maybe a curated list of  link to AWS docs would be better to keep this documentation simple ?

Reply:
Honestly the rendered version isn't that huge:

<img width="1026" alt="image" src="https://user-images.githubusercontent.com/720328/136025797-442f67d9-fa98-4fca-b363-51b7e267ebf4.png">

Ideally I'd like to shorten the sentences a bit, but I still think this is a net positive because that config is so cryptic (for newcomers) that it helps to have a bit of context. Even if it's a bit too much.

Reply:
Right, the links made it look like it was huge and it could indeed help newcomers :+1: 

At various places in bref docs, descriptions like this are inlined in the Cloudformation templates as comments which brings them closer to what it's describing. because this will likely be copied as-is in user projects, it will also be documented directly in their serverless.yml which can be handy for them.

And because we don't want whole paragraphs in YAML comments, it also helps keeping things small and simple. 

Then we can add links for further details. 

Something like this (written quickly, just to get an idea of what that could look like)

```yaml
        CacheTable:
            Type: 'AWS::DynamoDB::Table'
            Properties:
                AttributeDefinitions:
                    - AttributeName: id # adds a mandatory id field
                      AttributeType: S # the type of id will be a string
                KeySchema:
                    - AttributeName: id
                      KeyType: HASH # id will be the partition key (similar to primary keys in SQL)
                BillingMode: PAY_PER_REQUEST # billed for each request instead of paying for a fixed amount of operations
                TimeToLiveSpecification: # deletes cache keys automatically based on the ttl field which contains a timestamp
                    AttributeName: ttl
                    Enabled: true
```

Reply:
That indeed looks interesting! @RikudouSage see what you can do with those suggestions?

Reply:
I'll look into it all in an hour or so